src/goto-symex/goto_symex.h

```diff
@@ -251,9 +251,9 @@ class goto_symext
   // gotos
   void merge_gotos(statet &);
 
-  virtual void merge_goto(const goto_statet &goto_state, statet &);
+  virtual void merge_goto(goto_statet &goto_state, statet &);
```

Comment:
I'm not sure what's happening here because there is no documentation. If the `goto_state` is not supposed to be used afterwards then it would be good to make it clear, either by documenting or in the type, by forcing the argument to be movable.

Reply:
Fair, I'll work on adding documentation.

Reply:
Documentation added.